pkgs/applications/audio/nova-filters/default.nix

```diff
@@ -9,20 +9,22 @@ stdenv.mkDerivation {
     sha256 = "16064vvl2w5lz4xi3lyjk4xx7fphwsxc14ajykvndiz170q32s6i";
   };
 
-  nativeBuildInputs = [ pkg-config sconsPackages.scons_3_0_1 ];
+  nativeBuildInputs = [ pkg-config sconsPackages.scons_latest ];
   buildInputs = [ boost ladspaH ];
 
   patchPhase = ''
-    # remove TERM:
-    sed -i -e '4d' SConstruct
-    sed -i "s@mfpmath=sse@mfpmath=sse -I ${boost.dev}/include@g" SConstruct
-    sed -i "s@ladspa.h@${ladspaH}/include/ladspa.h@g" filters.cpp
-    sed -i "s@LADSPA_HINT_SAMPLE_RATE, 0, 0.5@LADSPA_HINT_SAMPLE_RATE, 0.0001, 0.5@g" filters.cpp
-    sed -i "s/= check/= detail::filter_base<internal_type, checked>::check/" nova/source/dsp/filter.hpp
+    # remove TERM and Options:
+    sed -i -e '4d;7,17d' SConstruct
```

Comment:
Can we do a patch instead? This could easily break with the next update.

Reply:
The package was from 2008 and never updated, so this should be fine...

Reply:
I've made the diff lighter.